adhocracy4/comments/static/comments/Comment.jsx

```diff
@@ -8,12 +8,15 @@ var CommentManageDropdown = require('./CommentManageDropdown')
 
 var React = require('react')
 var django = require('django')
-var moment = require('moment')
 
 var safeHtml = function (text) {
   return { __html: text }
 }
 
+var localeDate = function (dateStr) {
+  return new Date(dateStr).toLocaleString(document.documentElement.lang)
```

Comment:
According to spec, the `Date` constructor only supports a very strict date format. Browsers tend to be more lenient, but this can be a source for incompatibilities.

Reply:
I checked again and the indeed the [DRF documentation](http://www.django-rest-framework.org/api-guide/fields/#datetimefield-format-strings) already states that correct format is used. So we are fine here.